<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="Size" stdset="0">
   <string notr="true">10</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="setBackgroundColorButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Set background color...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setSelectionColorButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Set selection color...</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Detail level</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="renderDetailLevelSlider">
         <property name="minimum">
          <number>50</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="shadingLabel">
         <property name="text">
          <string>Unselected regions shading</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="shadingSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>111</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="anaglyphViewGroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Anaglyph view</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Eyes shift</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="eyesShiftSlider">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>199</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="glassesColorLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Glasses colors</string>
         </property>
        </widget>
        <widget class="QComboBox" name="glassesColorSchemeComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="leftEyeColorLabel">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Left</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="rightEyeColorLabel">
            <property name="text">
             <string>Right</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="leftEyeColorChangeButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="rightEyeColorChangeButton">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>72</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="swapColorsButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Swap colors</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>72</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>setBackgroundColorButton</tabstop>
  <tabstop>setSelectionColorButton</tabstop>
  <tabstop>renderDetailLevelSlider</tabstop>
  <tabstop>shadingSlider</tabstop>
  <tabstop>anaglyphViewGroupBox</tabstop>
  <tabstop>eyesShiftSlider</tabstop>
  <tabstop>glassesColorSchemeComboBox</tabstop>
  <tabstop>leftEyeColorChangeButton</tabstop>
  <tabstop>rightEyeColorChangeButton</tabstop>
  <tabstop>swapColorsButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>setBackgroundColorButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>sl_setBackgroundColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setSelectionColorButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>sl_setSelectionColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>renderDetailLevelSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>sl_setRenderDetailLevel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eyesShiftSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>sl_setEyesShift()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>anaglyphViewGroupBox</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>sl_setAnaglyph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>glassesColorSchemeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>sl_setGlassesColorScheme()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leftEyeColorChangeButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>sl_setLeftEyeColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rightEyeColorChangeButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>sl_setRightEyeColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>swapColorsButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>sl_swapColors()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shadingSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>sl_setShadingLevel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sl_setBackgroundColor()</slot>
  <slot>sl_setSelectionColor()</slot>
  <slot>sl_setAtomSize()</slot>
  <slot>sl_setRenderDetailLevel()</slot>
  <slot>sl_setAnaglyph()</slot>
  <slot>sl_setGlassesColorScheme()</slot>
  <slot>sl_setLeftEyeColor()</slot>
  <slot>sl_setRightEyeColor()</slot>
  <slot>sl_swapColors()</slot>
  <slot>sl_setEyesShift()</slot>
 </slots>
</ui>
